media-sound/audacity/audacity-3.2.2.ebuild

```diff
@@ -0,0 +1,231 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+WX_GTK_VER="3.2-gtk3"
+
+inherit cmake wxwidgets xdg
+
+MY_P="Audacity-${PV}"
+DOC_PV="${PV}"
+# A header-only thread pool library, without a build system, about 100
```

Comment:
Shouldn't this comment be directly above `MY_THREADPOOL=…`?

Reply:
hmm, must've gotten reordered by accident, but yes